public/static/docs/command-reference/import.md

```diff
@@ -15,36 +15,37 @@ import.
 usage: dvc import [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
 
 positional arguments:
-  url         URL of Git repository with DVC project to download from.
-  path        Path to data within DVC repository.
+  url         Location of DVC project or Git repository to download from
+  path        Path to a file or directory within the project or repository
 ```
 
 ## Description
 
-DVC provides an easy way to reuse datasets, intermediate results, ML models, or
-other files and directories tracked in another <abbr>DVC repository</abbr> into
-the workspace. The `dvc import` command downloads such a <abbr>data
-artifact</abbr> in a way that it is tracked with DVC, so it can be updated when
-the data source changes. (See `dvc update`.)
+Provides an easy way to reuse files or directories tracked in any <abbr>DVC
+project</abbr> (e.g. datasets, intermediate results, ML models) or Git
+repository (e.g. other files and directories), into the workspace. The
+`dvc import` command downloads such a <abbr>data artifact</abbr> in a way that
+it is tracked with DVC, so it can be updated when the data source changes. (See
+`dvc update`.)
```

Comment:
OK this is almost perfect actually, I'll just make some minor updates in a future PR.

Reply:
Done in 44e7fcac.